README.md

```diff
@@ -78,6 +78,19 @@ npm run start:all
 
 Then open http://localhost:3000/ in your browser and enjoy!
 
+### Run in a docker
+
+Build image with:
+
+```bash
+docker build -t pytorch_explorer 
```

Comment:
```suggestion
docker build -t pytorch_explorer .
```
or even
` sudo docker build -t pytorch_explorer . `
if the user is not in the sudoers file